src/eko/couplings.py

```diff
@@ -37,12 +37,48 @@ def couplings_mod_ev(mod_ev):
 
 @nb.njit(cache=True)
 def expanded_lo(ref, beta0, lmu):
+    """Compute expanded solution at LO.
+
+    Parameters
+    ----------
+        ref: float
+            reference value of the coupling
+        beta0: float
+            LO beta function
```

Comment:
it's not the "beta function" but the first coefficient of the function (of course here and below)